### PR TITLE
initial inlay component

### PR DESCRIPTION
**Pull request by nelix**
screen shot 2015-02-26 at 1 47 41 pm

# Inlay

Used for indicating the subject of a dialogs

### CODE REVIEW

File: packages/inlay/index.jsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  propTypes: {
+    children: React.PropTypes.node.isRequired,
+  },
+
+  render() {
+    return <span style={{
+        borderRadius: 3,
+        background: '#eee',
+        padding: 3,
+      }}>
```
Comment (grapegravity) [inline packages/inlay/index.jsx]: What’s your rationale for not using CSS?

File: packages/inlay/README.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Inlay
+
+Used for indicating the subject of a dialogs
```
Comment (grapegravity) [inline packages/inlay/README.md]: I know it doesn’t have any props, but it’d be nice to document _that_.

File: packages/inlay/index.jsx (hunk 1 above)
Comment (nelix) [inline packages/inlay/index.jsx]: What is the rationale for using it?

Comment (grapegravity) [inline packages/inlay/index.jsx]: Everything else we have is using CSS

File: packages/inlay/README.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+## Usage
+
+<Inlay>blah</Inlay>
```
Comment (grapegravity) [inline packages/inlay/README.md]: Not in a code block?

File: packages/inlay/index.jsx (hunk 1 above)
Comment (nelix) [inline packages/inlay/index.jsx]: does not seem important

**Pull request merged**